kong/db/schema/entities/wasm_filter_chains.lua

```diff
@@ -33,22 +33,36 @@ local filter = {
 return {
   name = "wasm_filter_chains",
   primary_key = { "id" },
-  endpoint_key = "name",
+  endpoint_key = "id",
```

Comment:
Service/route fields are now unique as of this PR. Since there can only ever be one filter chain entity associated with a given route or service, the `name` field just doesn't really feel like it serves much purpose.

I wouldn't be particularly opposed to putting it back in (it _is_ a pretty standard field for most core entities), but that was the rationale for removing it: metadata bloat.

Reply:
Okay, that makes sense now.
Here are some rationales for adding it back in:
- UIs generally like displaying names over IDs, they are a bit more self-documenting than a UUID that tells you nothing
- decK and declarative systems like KIC need a unique property that they/user can control and treat that as a primary key where possible; this is necessary to tie the declrattive state that these tools maintain with the state in the database (against which they reconcile the declarative configuration)
- the entities that don't have names (or rather endpoint keys) yet are certificates and ca-certificates - we should try to follow the laid down practice
- routes originally didn't have names originally because it was deemed as unnecessary, we had to later add in because of high user demand